flakyness.org

```diff
@@ -1,7 +1,17 @@
-* Flaky tests
+#+TITLE: Iroha 2 flaky test document
 
-** test: integration::triggers::event_trigger::test_mint_asset_when_new_asset_definition_created (25/100)
+This document serves to test and track changes to the flaky-ness of the
+tests used in the Iroha 2 CI pipeline.
 
-** test: multiple_networks stdout (1/100)
+This file should be updated after any performance critical changes
+only. In particular because most flaky tests are of the =integration=
```

Comment:
```suggestion
This file should only be updated after any performance critical changes.
In particular, because most flaky tests are of the =integration=
```